### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by pchampin**
README.md now simply points to the homepage.

The implementation list has been converted to HTML, and is now pointed to by the homepage.

Adresses part of #150

### CODE REVIEW

**Comment by afs** [comment]
Are there two purposes here now?

The original purpose was "systems have been adding extensions".

We now have adoption of "RDF-star" in the EARL reports

This could be expressed either as a extra column linking to received reports, or two tables with text explaining.

**Comment by pchampin** [comment]
> We now have adoption of "RDF-star" in the EARL reports

true! :)

I propose to replace the "source" link by "implementation report" for those implementation that have one.

**Review by gkellogg — APPROVED** [review]

**Comment by TallTed** [comment]
The new unordered list provides less information, and is harder to digest, than the old table. I suggest putting the info back into a table. It may be appropriate to adjust the columns to match the evolved/evolving spec, eliminating display/mention of PG and SA modes, and instead talking about the notations and support (or lack thereof) for "unasserted embedded triples" and the like...

**Comment by pchampin** [comment]
@TallTed that better?

File: implementations.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          &nbsp;
+          &nbsp;
+          <img src="https://www.w3.org/Icons/SW/sw-cube.png" alt="Semweb Cube"/>
+        </p>
+        <div class="page-header">
+          <h1>RDF-star Implementations</h1>
+        </div> 	
+              
+        <p>To update this list, please submit a pull request to the <a href="https://github.com/w3c/rdf-star/blob/main/implementations.html">Github repository</a>, or contact the public mailing list (public-rdf-star@w3.org).</p>
+
```
Comment (gkellogg) [inline implementations.html:52]:
```suggestion
        <p>For the list of submitted implementation reports see <a href="https://w3c.github.io/rdf-star/reports/">RDF-star Processor Conformance</a>.</p>

```

File: implementations.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          <tr><td>Corese
+            <td><a href="http://ns.inria.fr/sparql-extension/rdfstar.html">documentation</a>
+            <td><a href="https://w3c.github.io/rdf-star/rdf-star-cg-spec.html#sa-mode-and-pg-mode">PG mode</a>
+          <tr><td>EYE
+            <td><a href="https://w3c.github.io/rdf-star/reports/#subj_EYE_Prolog">implementation report</a>
+            <td>
+          <tr><td>GraphDB
+            <td><a href="http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html">documentation</a>
+            <td>
+          <tr><td>Jena
```
Comment (afs) [inline implementations.html:74]:
```suggestion
          <tr><td>Apache Jena
```

File: implementations.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            <td><a href="https://jena.apache.org/documentation/rdfstar/">documentation</a>
+            <td>
+          <tr><td>rdf4j
```
Comment (afs) [inline implementations.html:77]:
```suggestion
          <tr><td>Eclipse rdf4j
```

**Review by afs — CHANGES_REQUESTED** [review]

**Pull request merged**